docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
is this file needed somewhere?

Reply:
I though it would be necessary but I think it is actually not. I removed it.